### ISSUE TITLE
Relationship Picker filtering is broken

### ISSUE THREAD

**Issue by AnthonyCretenet** [Budibase/budibase#14067]
**Checklist**
- [Y] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: Docker compose
  - Budibase Version: 2.29.5

**Describe the bug**
When I use a Relationship Picker and want to use the filters, it's not possible.
When I click on the _No filters set_ button to define a filter, nothing opens and the Budibase interface freezes. So I can't set any filter.

![Capture d'écran 2024-07-02 105758](https://github.com/Budibase/budibase/assets/174427987/671e883c-0ce7-4997-980b-3a72f771e343)

**To Reproduce**
1. Create a new form
2. Add a Relationship picker
3. Add a filter

**Desktop**
 - OS: Microsoft Windows 11
 - Browser: Mozilla Firefox
 - Version: 127.0.2

**Comment by ConorWebb96** [comment]
console error

```
index-64e0667e.js:397 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'tableId')
    at index-64e0667e.js:397:48841
    at Array.find (<anonymous>)
    at Gr.$$.update (index-64e0667e.js:397:48820)
    at init$2 (index-64e0667e.js:4:11494)
    at new FilterBuilder (index-64e0667e.js:397:49649)
    at create_fragment$65 (index-64e0667e.js:1596:13906)
    at init$2 (index-64e0667e.js:4:11550)
    at new FilterBuilder_1 (index-64e0667e.js:1596:15702)
    at Array.create_default_slot$2H (index-64e0667e.js:1640:56591)
    at create_slot (index-64e0667e.js:1:1002868)
(anonymous) @ index-64e0667e.js:397
Gr.$$.update @ index-64e0667e.js:397
init$2 @ index-64e0667e.js:4
FilterBuilder @ index-64e0667e.js:397
create_fragment$65 @ index-64e0667e.js:1596
init$2 @ index-64e0667e.js:4
FilterBuilder_1 @ index-64e0667e.js:1596
create_default_slot$2H @ index-64e0667e.js:1640
create_slot @ index-64e0667e.js:1
create_fragment$9N @ index-64e0667e.js:185
init$2 @ index-64e0667e.js:4
DrawerContent @ index-64e0667e.js:185
create_body_slot$7 @ index-64e0667e.js:1640
create_slot @ index-64e0667e.js:1
create_default_slot$62 @ index-64e0667e.js:175
create_slot @ index-64e0667e.js:1
create_fragment$aV @ index-64e0667e.js:4
init$2 @ index-64e0667e.js:4
Portal$1 @ index-64e0667e.js:4
create_if_block$4z @ index-64e0667e.js:175
p @ index-64e0667e.js:175
update @ index-64e0667e.js:4
flush @ index-64e0667e.js:4
Promise.then (async)
schedule_update @ index-64e0667e.js:4
make_dirty @ index-64e0667e.js:4
(anonymous) @ index-64e0667e.js:4
(anonymous) @ index-64e0667e.js:175
Qr @ index-64e0667e.js:4
observe$3 @ index-64e0667e.js:175
fo @ index-64e0667e.js:182
(anonymous) @ index-64e0667e.js:1640
(anonymous) @ index-64e0667e.js:4
bubble @ index-64e0667e.js:4
bo @ index-64e0667e.js:74
(anonymous) @ index-64e0667e.js:1
```

**Comment by Socodevi** [comment]
it is broken on my installation too, which led me here to this github issue post.
version 2.29.12
database Mariadb

**Comment by Reftek85** [comment]
it is broken on my installation too, which led me here to this github issue post.
version 2.29.12
database Mariadb

**Comment by ConorWebb96** [comment]
The fix for this is [here](https://github.com/Budibase/budibase/pull/14109). A new release of Budibase just needs pushed and this should then be fixed.

Likely to be fixed in `2.29.7`
